### PR TITLE
Updated css for sync-with-system tooltips.

### PR DESCRIPTION
**Pull request by mateuszzagorski**
After the recent editor update to v 35.2.1 it was necessary to update css for tooltips when using `sync-with-system` theme. Closing #390.

### CODE REVIEW

**Review by mmotyczynska — CHANGES_REQUESTED** [review]
Besides the comment regarding changing the tooltip background, there is still one issue we have to address. The triangle in the toolbar tooltips does not look good I guess in all theme variants. We recently solved a similar problem for [tooltips for image/table](https://github.com/ckeditor/github-writer/pull/400) so maybe you could find there some inspiration for this fix.

File: src/app/theme/panel.css [hunk 1]
```diff
@@ -60,3 +60,8 @@
 html[data-color-mode="light"] .ck.ck-balloon-panel.ck-balloon-panel_arrow_n.ck-balloon-panel_with-arrow.ck-tooltip {
 	background-color: var(--ck-color-tooltip-background) !important
 }
+
+/* Updating tooltip's text color for the 'sync-with-system' theme. See #390. */
+html[data-color-mode="auto"] .ck.ck-balloon-panel.ck-tooltip .ck-tooltip__text {
+	color: var(--ck-color-tooltip-background)
+}
```
[the remark this answers is not in the document]
Comment (mateuszzagorski) [reply src/app/theme/panel.css:67]: So after adding some css and a bit of tweaking of the styles we already had, it should be unified now and look like this:

<table><tbody><tr><td>sync-with-system (light)</td><td>Sync-with-system (dark)</td><td>default (light)</td><td>default (dark)</td></tr><tr><td><figure class="image"></figure></td><td><figure class="image"></figure></td><td><figure class="image"></figure></td><td><figure class="image"></figure></td></tr></tbody></table>

**Review by mmotyczynska — APPROVED** [review]
Looks good now :+1:

File: src/app/theme/panel.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (mmotyczynska) [reply src/app/theme/panel.css]: OK :+1:

**Pull request merged**